### ISSUE TITLE
Non-zero hierarchy origin for some single panel detector models

### ISSUE THREAD

**Issue by dagewa** [cctbx/dxtbx#472]
Reported by a user and seen to affect at least MRC format data. I see, for example:

```
$ grep "origin" -A3 cbz-4/imported.expt 
          "origin": [
            -28.6720009,
            28.6720009,
            -943.0
--
        "origin": [
          0.19599998470000202,
          -0.9240000081000019,
          0.0
```
This is weird. The origin reported by `dials.show` seems to be the vector sum of these two. After refinement:
```
$ grep "origin" -A3 cbz-4/refined.expt 
          "origin": [
            3.739881805363169,
            28.23693830418512,
            -943.0000000000009
--
        "origin": [
          0.19599998470000202,
          -0.9240000081000019,
          0.0
```
so the first origin is updated, while the second (the hierarchy origin) remains the same. Why does this happen in `FormatMRC`, and does it affect any other formats?

**Comment by dagewa** [comment]
Construction looks innocuous: https://github.com/cctbx/dxtbx/blob/1d2d38e58464b29e0d886ec875bd94f5cc85be82/src/dxtbx/format/FormatMRC.py#L204-L213

**Comment by biochem-fan** [comment]
Didn't he or she use `fast_slow_beam_centre` in `dials.import`?

This keeps the origin of the panel the same and put the residual to the top hierarchy.

**Comment by dagewa** [comment]
Aha! Thank you. This is probably the case. Certainly, it is true for my example
```
$ grep fast_ cbz-4/dials.import.log 
    fast_slow_beam_centre = 1017 991
```
This behaviour does seem rather weird though. :thinking:

**Comment by biochem-fan** [comment]
Indeed this is weird for single panel detectors. For multipanel detectors like CSPAD and MPCCD, the current behavior might make sense (keep individual panels fixed and move the whole detector around).

**Comment by dagewa** [comment]
Agreed. I updated the title accordingly. I would rather have the code move the `Panel` for single panel detectors, and keep the behaviour as-is for multipanel - as you suggest.
